engine/graphics/src/dmsdk/graphics/graphics.h

```diff
@@ -34,16 +34,21 @@ namespace dmGraphics
      * @typedef
      * @name HContext
      */
-    typedef struct Context* HContext;
+    typedef void* HContext;
```

Comment:
Maybe not 100% necessary, but It's really hard to debug when it's defined as a struct ptr and linking with multiple graphics adapters.